mofacts/client/views/experiment/unitEngine.js

```diff
@@ -1546,17 +1539,7 @@ function scheduleUnitEngine() {
       return settings;
     }
 
-    const rawAssess = _.safefirst(unit.assessmentsession);
```

Comment:
I'm not saying this is wrong, but what's the tradeoff being done here by refactoring this?

Reply:
Since the tdf parameters dont come as single length arrays parsing them is no longer necessary. That was the point of this part of the code from what I saw.